libs/wordpress-plugin/README.md

```diff
@@ -0,0 +1,103 @@
+## Getting started
+
+This repository is for contributing to the WordPress plugin.
+
+If you are looking to install it on your own WordPress server, please visit https://blockprotocol.org/wordpress
+
+## Working on the plugin
+
+### First time set up
+
+- run `yarn`
+- run `yarn dev:wordpress`
+- run `yarn dev:plugin`
+- visit [http://localhost:8000](http://localhost:8000)
+- go through the WordPress set up flow
+- go to Plugins in the sidebar, activate Block Protocol
+- go to Block Protocol Settings in the sidebar, enter an API key
+
+### Regular development
+
+- `yarn dev:plugin` to run the plugin
+- Optional: `yarn dev:wordpress` (to attach to Docker – useful for inspecting server logs)
```

Comment:
Is this step optional? It also starts the wordpress instance so it should be run always, right?
```suggestion
- `yarn dev:wordpress` (WP server logs are visible in the output)
```